### PR TITLE
CNV18218: Add migration progress popover

### PR DESCRIPTION
**Pull request by avivtur**
Signed-off-by: Aviv Turgeman <aturgema@redhat.com>
## 📝 Description

Adding a popover to display the migration progress, can be found in the single VM overview details page in details card (Status field when VM printable status is `Migrating`  

[UX design](https://docs.google.com/document/d/1bkWARQa_l1NsSF4PdLt2ctMUiVjNxYk9QXwFLjybKc4/edit#)
[Issue](https://issues.redhat.com/browse/CNV-18218)

## 🎥 Demo

https://user-images.githubusercontent.com/67270715/187198300-3d06b906-7139-44d1-b5f1-1108d2aa0dcf.mp4

### CODE REVIEW

File: src/views/virtualmachines/utils/virtualMachineStatuses.ts [hunk 1]
```diff
@@ -36,33 +37,29 @@ export const errorPrintableVMStatus = {
...
+  if (status === printableVMStatus.Running) {
+    return SyncAltIcon;
+  }
+  if (status === printableVMStatus.Stopped) {
+    return OffIcon;
+  }
+  if (status === printableVMStatus.Migrating) {
+    return MigrationIcon;
+  }
+  if (status?.toLowerCase().includes('error')) {
```
Comment (pcbailey) [inline src/views/virtualmachines/utils/virtualMachineStatuses.ts:53]: This won't catch `ErrImagePull`. Why not make this a switch statement?

Comment (avivtur) [reply src/views/virtualmachines/utils/virtualMachineStatuses.ts:53]: you're right, this logic was copied from 4.10 where we did things a bit different, changed to switch case and to catch all error statuses, thanks :)

File: locales/en/plugin__kubevirt-plugin.json [hunk 2]
```diff
@@ -27,6 +27,7 @@
   "{{dsLabel}} name": "{{dsLabel}} name",
   "{{dsLabel}} project": "{{dsLabel}} project",
   "{{matchingNodeText}} matching": "{{matchingNodeText}} matching",
+  "{{minutes}}{{seconds}} seconds": "{{minutes}}{{seconds}} seconds",
```
[the remark this answers is not in the document]
Comment (avivtur) [reply locales/en/plugin__kubevirt-plugin.json:30]: no the point is to print this pattern: `x minutes, y seconds` if x is 0 don't show the minutes section at the beginning, [here](https://github.com/kubevirt-ui/kubevirt-plugin/pull/850/files#diff-be4673eedfaa3814fd53048d9902ab112eae1b2a7d567cc219b044777c430907R67-R70)

**Pull request merged**